src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main>
+      <ul class="calendar calendar__month-length--29">
+        <li class="calendar__item calendar__item--1 calendar__item--start-day"></li>
+        <li class="calendar__item calendar__item--2"></li>
+        <li class="calendar__item calendar__item--3"></li>
+        <li class="calendar__item calendar__item--4"></li>
+        <li class="calendar__item calendar__item--5"></li>
+        <li class="calendar__item calendar__item--6"></li>
+        <li class="calendar__item calendar__item--7"></li>
+        <li class="calendar__item calendar__item--8"></li>
+        <li class="calendar__item calendar__item--9"></li>
+        <li class="calendar__item calendar__item--10"></li>
+        <li class="calendar__item calendar__item--11"></li>
+        <li class="calendar__item calendar__item--12"></li>
+        <li class="calendar__item calendar__item--13"></li>
+        <li class="calendar__item calendar__item--14"></li>
+        <li class="calendar__item calendar__item--15"></li>
+        <li class="calendar__item calendar__item--16"></li>
+        <li class="calendar__item calendar__item--17"></li>
+        <li class="calendar__item calendar__item--18"></li>
+        <li class="calendar__item calendar__item--19"></li>
+        <li class="calendar__item calendar__item--20"></li>
+        <li class="calendar__item calendar__item--21"></li>
+        <li class="calendar__item calendar__item--22"></li>
+        <li class="calendar__item calendar__item--23"></li>
+        <li class="calendar__item calendar__item--24"></li>
+        <li class="calendar__item calendar__item--25"></li>
+        <li class="calendar__item calendar__item--26"></li>
+        <li class="calendar__item calendar__item--27"></li>
+        <li class="calendar__item calendar__item--28"></li>
+        <li class="calendar__item calendar__item--29"></li>
+        <li class="calendar__item calendar__item--30"></li>
+        <li class="calendar__item calendar__item--31"></li>
```

Comment:
Since the rest of the scss files are missing I'm not sure about these extra modifiers like calendar__item--1(up until 31), you need to add the numbers with the ::before.